package.json

```diff
@@ -19,20 +19,21 @@
   },
   "build": {
     "appId": "org.ssbc.patchbay",
-    "asarUnpack": [
-      "node_modules/sodium-native/prebuilds"
-    ],
-    "npmSkipBuildFromSource": true,
     "npmArgs": [
       "--abi=57"
     ],
     "files": [
       "**/*",
       "!node_modules/**/deps${/*}",
-      "!node_modules/**/_node_modules${/*}",
-      "!node_modules/sodium-native/prebuilds${/*}",
-      "node_modules/sodium-native/prebuilds/${os}-${arch}/electron-57.node",
-      "node_modules/sodium-native/prebuilds/${os}-${arch}/libsodium.*"
+      "!node_modules/**/_node_modules${/*}"
+    ],
+    "extraFiles": [
+      "app/node_modules/sodium-native/prebuilds/${os}-${arch}/electron-57.node",
+      "app/node_modules/sodium-native/prebuilds/${os}-${arch}/libsodium.*",
```

Comment:
can you help me get my head around why it's safe to move these prebuilds from `files` to `extraFiles` @cryptix ? 
What's the difference ... I read the docs and found it kinda opaque